docs/topics/windows-provisioning-scripts.md

```diff
@@ -0,0 +1,29 @@
+# Windows Provisioning Scripts
+
+## Overview
+
+Due to requirements around signing executable code on Windows nodes some powershell scripts are are delivered via a .zip file during node provisioning.
+This document details how to update and test changes to these files.
+
+## Testing
+
+Changes to the provisioning scripts packaged in the .zip can be tested by:
+
+- Running [scripts/build-windows-provisioning-scripts.sh](../../scripts/build-windows-provisioning-scripts.sh) which will create a .zip and upload it to a storage account
+- Create/update a cluster definition file and set `windowsProfile.provisioningScriptsPackageURL' to point to the uploaded back.
+- Perform a cluster deployment with aks-engine.
+
+## Updates
```

Comment:
Are these steps automated or are these the steps that someone would need to automate if making there own updates?

Reply:
These steps are not currently automated.
Automating this is not necessary for testing their own updates - but might be helpful.